src/com/facebook/buck/android/ApkBuilderStep.java

```diff
@@ -187,6 +187,14 @@ private PrivateKeyAndCertificate createKeystoreProperties()
     String alias = keystoreProperties.getAlias();
     char[] keyPassword = keystoreProperties.getKeypass().toCharArray();
     Key key = keystore.getKey(alias, keyPassword);
+    // key can be null if alias/password is incorrect.
+    if (key == null) {
+      throw new HumanReadableException(
+          "The keystore [%s] key.alias [%s] does not exist or does not identify a key-related entry",
```

Comment:
`ant lint` fails here because the line is too long.  Travis would have told you this if Travis was working properly...